docs/contributor/01-00-installation.md

```diff
@@ -0,0 +1,241 @@
+```
+
+This command builds a Docker image for Istio Operator and pushes it to the registry and path, as defined in `IMG_REGISTRY`.
+
+6. Verify that the Istio module's image and Istio Operator's image are pushed to the local registry. Run:
+
+```bash
+curl registry.localhost:5001/v2/_catalog
+```
+If the images were pushed successfully, you get the following output:
+
+```json
+{
+    "repositories": [
+        "unsigned/component-descriptors/kyma.project.io/module/istio",
+        "unsigned/operator-images/istio-operator"
+    ]
+}
+```
+
+7. Inspect the generated module template. Change the existing repository context in `spec.descriptor.component`:
+
+```yaml
+repositoryContexts:                                                                           
+- baseUrl: registry.localhost:5000/unsigned                                                   
+  componentNameMapping: urlPath                                                               
+  type: ociRegistry
+```
+>**NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. k3d has registry aliases but module-manager is not part of k3d and thus does not know how to properly alias `registry.localhost:5001`
+
+>**TIP:** Apply `"operator.kyma-project.io/use-local-template": "true"` to make sure that Lifecycle Manager uses the registry URL present in the ModuleTemplate.
+
+### Use Lifecycle Manager to install Istio in modular Kyma on k3d
+
+1. Install the latest version `lifecycle-manager`.
+
+```bash
+kyma alpha deploy
+```
+If deployed successfully, you get the following output:
+
+```bash
+- Kustomize ready
+- Lifecycle Manager deployed
+- Module Manager deployed
+- Modules deployed
+- Kyma CR deployed
+- Kyma deployed successfully!
+
+Kyma is installed in version:
+Kyma installation took: 18 seconds
+
+Happy Kyma-ing! :)
+```
+
+Kyma installation is ready, but no module is activated yet.
+
+```bash
+kubectl get kymas.operator.kyma-project.io -A
+NAMESPACE    NAME           STATE   AGE
+kcp-system   default-kyma   Ready   71s
+```
+
+2. Apply `template.yaml` to register Istio as a module known for Lifecycle Manager.
+
+Istio Module is a known module, but it is not activated.
+
+```bash
+kubectl get moduletemplates.operator.kyma-project.io -A 
+NAMESPACE    NAME                  AGE
+kcp-system   moduletemplate-istio   2m24s
+```
+
+3. Give Lifecycle Manager permission to install CustomResourceDefinition (CRD) cluster-wide.
+
+>**NOTE:** This is a temporary workaround only required in the single-cluster mode.
+
+Module-manager must be able to apply CRDs to install modules. When it operates in remote mode, where the control-plane manages remote clusters, it receives an administrative kubeconfig that allows it to target the remote cluster and apply CRDs. However, when it operates in local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
```

Comment:
Do we need the hyphens here?
There is `module-manager` at the beginning of this section and' module manager' in the next paragraph. Is there a difference?
```suggestion
Module manager must be able to apply CRDs to install modules. When it operates in remote mode, where the control plane manages remote clusters, it receives an administrative kubeconfig that allows it to target the remote cluster and apply CRDs. However, when it operates in local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
```